src/core/rateLimiter.js

```
@@ -1,270 +1,343 @@
-        }
-        
-        return true;
+    this.requestCounts = {
+      github: { count: 0, resetTime: null },
+      gitlab: { count: 0, resetTime: null },
+    };
+  }
+
+  // Set API token - updateHeader logic
+  setToken(token, platform) {
+    if (!this.tokens.hasOwnProperty(platform)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)